README.md

```diff
@@ -167,6 +167,8 @@ export default withSession(handler, options);
 | name | The name of the cookie to be read from the request and set to the response. | `sid` |
 | store | The session store instance to be used. | `MemoryStore` |
 | genid | The function that generates a string for a new session ID. | [`nanoid`](https://github.com/ai/nanoid) |
+| encodeId | Transforms generated session id before setting cookie. Useful for things like signing. |
+| decodeId | Transforms session id back while getting from cookie |
```

Comment:
encode and decode*

Reply:
You also missed the default values column.

Reply:
You mean encode/decode for the name of the options/methods or instead of the word "transforms" in the description? I though encodeId/encodeId might be more specific instead of asking, "encode what?". I was also unsure about capitalization since most follow camelCase, but not genid. I can make the change either way, so lemme know

Reply:
And apologies on the default column. I will set them as `undefined` or should I use `unset` since that is used below for `cookie.domain` and `cookie.sameSite`

Reply:
Oh I see my mistake about encode/decode vs encodeId/decodeId: The actual code uses encode/decode while I documented encodeId/decodeId. My apologies again. I will take suggestions and fix accordingly

Reply:
Based on what @hoangvvo said [here](https://github.com/hoangvvo/next-session/pull/110#issuecomment-612971000). I think it would be encode/decode, because we can use it to encrypt and decrypt the cookie also.

Reply:
Makes sense. I will wait for a little bit to see if wants to weigh in anyway then go ahead with encode/decode

Reply:
As an example, so far we have:
```
| encode | Transforms session before setting cookie. Useful for things like signing or encrypting. | undefined |
| decode | Transforms session back while getting from cookie | undefined |
```

Reply:
@ealtunyay that’s a really good description. I would change session to session ID